adapters/mabidder/mabidder.go

```diff
@@ -0,0 +1,112 @@
+package mabidder
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type maServerResponse struct {
+	Responses       []maBidResponse
+	PrivateIdStatus string `json:"-"`
+}
+
+type maBidResponse struct {
+	RequestID         string  `json:"requestId"`
+	Currency          string  `json:"currency"`
+	Width             int32   `json:"width"`
+	Height            int32   `json:"height"`
+	PlacementId       string  `json:"creativeId"`
+	Deal              string  `json:"dealId,omitempty"`
+	NetRevenue        bool    `json:"netRevenue"`
+	TimeToLiveSeconds int32   `json:"ttl"`
+	AdTag             string  `json:"ad"`
+	MediaType         string  `json:"mediaType"`
+	Meta              maMeta  `json:"meta"`
+	CPM               float32 `json:"cpm"`
+}
+
+type maMeta struct {
+	AdDomain []string `json:"advertiserDomains"`
+}
+
```

Comment:
scope of these struct is within `mabidder`  package. so could `ma` prefix from above structs
```
- type maServerResponse struct {
+ type serverResponse struct {

- type maBidResponse struct {
+ type bidResponse struct {

-  type maMeta struct {
+  type meta struct {
```